### DISCUSSION TITLE
G6 5.x版本实现 节点拖动后，固定位置，拖动其他节点的时候，不参与移动，类似于4.x版本

### DISCUSSION THREAD

**Discussion by dnwu** [antvis/G6#6655] [Q&A]
4.x的demo这样子https://g6-v4.antv.vision/examples/net/forceDirected/#basicForceDirectedDragFix

**Comment by petercat-assistant** [comment]


**Reply by dnwu** [reply]
该方法不行

**Reply by dnwu** [reply]
我是用的是d3-force布局

**Comment by Aarebecca** [comment]
@dnwu 可以参考这个交互实现，调用布局 setFixedPosition 方法固定位置

> https://github.com/antvis/G6/blob/v5/packages/g6/src/behaviors/drag-element-force.ts

本质上是设置 fx,fy 值，d3-force会读取这个值
